package-lock.json

```diff
@@ -1,8 +1,2340 @@
 {
   "name": "MovieCollections",
```

Comment:
co to za zmiana?

Reply:
Cwiekala Michal, 3 weeks ago   (December 6th, 2021 7:25am) 

added NPM deployment to gh pages